src/test/java/org/codeforamerica/messaging/TestData.java

```diff
@@ -16,35 +15,29 @@ public class TestData {
     public static final String TEMPLATE_NAME = "default template name";
     public static final String TEMPLATE_SUBJECT_DEFAULT = "English A Subject: {{placeholder}}";
     public static final String TEMPLATE_BODY_DEFAULT = "English A Body: {{placeholder}}";
-    public static final String TEMPLATE_SUBJECT_EN_B = "English B Subject: {{placeholder}}";
-    public static final String TEMPLATE_BODY_EN_B = "English B Body: {{placeholder}}";
-    public static final String TEMPLATE_SUBJECT_ES_A = "Spanish A Subject: {{placeholder}}";
-    public static final String TEMPLATE_BODY_ES_A = "Spanish A Body: {{placeholder}}";
     public static final String TEMPLATE_SUBJECT_ES_B = "Spanish B Subject: {{placeholder}}";
     public static final String TEMPLATE_BODY_ES_B = "Spanish B Body: {{placeholder}}";
 
-    public static TemplateVariant aDefaultTemplateVariant(Template template) {
-        return template.addTemplateVariant(TemplateVariant.builder()
-                .subject(TEMPLATE_SUBJECT_DEFAULT)
-                .body(TEMPLATE_BODY_DEFAULT));
+    public static TemplateVariant.TemplateVariantBuilder aTemplateVariant() {
```

Comment:
This should be renamed as one would expect `aTemplateVariant` to be a `TemplateVariant` and not a `TemplateVariant.TemplateVariantBuilder`.

Reply:
This a convention I saw used in a few blogs about using the Test Data Builder pattern. The idea is that you can modify the builder that is returned or keep the values that were already set depending on how you are using it, and don't have to set the values you don't care about. 
* http://www.natpryce.com/articles/000714.html
* https://www.arhohuttunen.com/test-data-builders/
* https://betterprogramming.pub/why-you-should-use-test-data-builders-714eb9de20c1

I saw some suggestions to change the lombok defaults when setting values to make it more semantically fluid but I didn't think it was needed. (e.g. "A template variant with treatment B")